glide.yaml

```diff
@@ -19,14 +17,16 @@ import:
 - package: github.com/projectcalico/go-json
   subpackages:
   - json
-- package: github.com/coreos/etcd
-  version: 17ae440991da3bdb2df4309936dd2074f66ec394
+- package: golang.org/x/sys
+  subpackages:
+  - unix
+  - windows
 - package: github.com/projectcalico/go-yaml-wrapper
 - package: github.com/projectcalico/libcalico-go
-  version: 4db56aa2f763478265b424ef04988cc6ddd30537
+  version: bb0e4b2ddbdc73d3718fd61d7e3f5b17216a5dca
```

Comment:
can we just leave this as master for now since it'll make revving glide.lock easier?